mm/2s2h/GameInteractor/GameInteractor.h

```diff
@@ -63,6 +63,7 @@ typedef enum {
     VB_CLOCK_TOWER_OPENING_CONSIDER_THIS_FIRST_CYCLE,
     VB_DRAW_SLIME_BODY_ITEM,
     VB_ZTARGET_SPEED_CHECK,
+    GI_VB_THIEF_BIRD_STEAL,
```

Comment:
We recently dropped the `GI_` prefix to all these enums, just need to update it here and the other files.

```suggestion
    VB_THIEF_BIRD_STEAL,
```

Reply:
Dropped the GI_